src/components/molecules/Settings/Project/PublishSection/index.tsx

```diff
@@ -71,15 +70,19 @@ const PublishSection: React.FC<Props> = ({
   return (
     <>
       <Wrapper>
-        <Section title={intl.formatMessage({ defaultMessage: "Site settings" })}>
+        <Section title={intl.formatMessage({ defaultMessage: "Domain Settings" })}>
           <StyledText size="m" color={theme.main.strongText}>
-            {intl.formatMessage({ defaultMessage: "Site name" })}
+            {intl.formatMessage({ defaultMessage: "Default Domain" })}
           </StyledText>
           <Text size="s">
-            {intl.formatMessage({
-              defaultMessage:
-                "By default, once published your project will be accessible via the URL below.",
-            })}
+            {alias
+              ? intl.formatMessage({
+                  defaultMessage: "Access your project, copy or edit the URL below.",
+                })
+              : intl.formatMessage({
+                  defaultMessage:
+                    "Once your project is published from the Editor page the URL details will be shown here.",
```

Comment:
```suggestion
                    "Once your project is published from the editor page the URL details will be shown here.",
```

Reply:
The reason I made it uppercase was @HideBa me and a couple others had a terminology meeting and discussed naming for sections of the app and we decided the "EarthEditor" would be referred to as the editor page for users(ie user guide, etc). If it is a name it can be capitalized. But for now I'll change it to lower case